docs/guide/migration.rst

```diff
@@ -9,4 +9,184 @@ This is a guide to migrate from Stable-Baselines to Stable-Baselines3.
+Default hyperparameters are taken from the nature paper, except for the optimizer and learning rate that were taken from Stable Baselines defaults.
+
+DDPG
+^^^^
+
+DDPG now follows the same interface as SAC/TD3.
+For state/reward normalization, you should use ``VecNormalize`` as for all other algorithms.
+
+SAC/TD3
+^^^^^^^
+
+SAC/TD3 now accept any number of critics, e.g. ``policy_kwargs=dict(n_critics=3)``, instead of only two before.
+
+
+.. note::
+
+	SAC/TD3 default hyperparameters (including network architecture) now match the ones from the original papers.
+	DDPG is using TD3 defaults.
+
+
+New logger API
+--------------
+
+- Methods were renamed in the logger:
+
+  - ``logkv`` -> ``record``, ``writekvs`` -> ``write``, ``writeseq`` ->  ``write_sequence``,
+  - ``logkvs`` -> ``record_dict``, ``dumpkvs`` -> ``dump``,
+  - ``getkvs`` -> ``get_log_dict``, ``logkv_mean`` -> ``record_mean``,
+
+
+Internal Changes
+----------------
+
+Please read the :ref:`Developper Guide <developer>` section.
+
+
+New Features
+============
+
+- much cleaner and consistent base code (and no more warnings =D!) and static type checks
+- independent saving/loading/predict for policies
+- A2C now supports Generalized Advantage Estimation (GAE) and advantage normalization (both are deactivated by default)
+- generalized State-Dependent Exploration (gSDE) exploration is available for A2C/PPO/SAC. It allows to use RL directly on real robots (cf https://arxiv.org/abs/2005.05719)
+- proper evaluation (using separate env) is included in the base class (using ``EvalCallback``),
+  if you pass the environment as a string, you can pass ``create_eval_env=True`` to the algorithm constructor.
+- better saving/loading: optimizers are now included in the saved parameters and there is two new methods ``save_replay_buffer`` and ``load_replay_buffer`` for the replay buffer when using off-policy algorithms (DQN/DDPG/SAC/TD3)
+- you can pass ``optimizer_class`` and ``optimizer_kwargs`` to ``policy_kwargs`` in order to easily
+  customize optimizers
+- seeding now works properly to have deterministic results
+- replay buffer does not grow, allocate everything at build time (faster)
+- we added a memory efficient replay buffer variant (pass ``optimize_memory_usage=True`` to the constructor), it reduces drastically the memory used especially when using images
+- you can specify an arbitrary number of critics for SAC/TD3 (e.g. ``policy_kwargs=dict(n_critics=3)``)
+
+
+How to migrate?
+===============
+
+In most cases, replacing ``from stable_baselines`` by ``from stable_baselines3`` will be sufficient.
+Some files were moved to the common folder (cf above) and could result to import errors.
+We recommend looking at the `rl-zoo3 <https://github.com/DLR-RM/rl-baselines3-zoo>`_ and compare the imports
+to the `rl-zoo <https://github.com/araffin/rl-baselines-zoo>`_ of SB2 to have a concrete example of successful migration.
+
+Planned Features
```

Comment:
Not really needed here. We will forget to update this in any case. Better to only have it in one place (e.g. readme or the issues)